requirements.txt

```diff
@@ -5,3 +5,4 @@ djangorestframework==3.9.0
 PyJWT==1.7.1
 six==1.12.0 
 psycopg2-binary==2.7.6.1
+pycodestyle-2.4.0
```

Comment:
Please include instructions for how to use pycodestyle in the README

Reply:
Okay, I'll include that in the readme